.profile

```diff
@@ -157,7 +159,7 @@ if [[ $MIGRATE_DB = 'True' ]]; then
   # Run migrations
   ckan db upgrade
   ckan harvester initdb
-  ckan archiver init
-  ckan report initdb
-  ckan qa init
+  # ckan archiver init
```

Comment:
where do we init archiver now?

Reply:
This commit is already in ckan-210. https://github.com/GSA/catalog.data.gov/commit/f7c28fea7b5042a69fb1ddc06cd57ac78633c3be
I am surprised it shows up here in the change list when I do rebasing main.